### PR TITLE
[16] footer component

### PR DESCRIPTION
**Pull request by Loczi94**

### CODE REVIEW

File: frontend/src/components/PageFooter.vue [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<template>
+    <div>
```
Comment (Bosmanfrx) [inline frontend/src/components/PageFooter.vue]: Why encapsulate footer in div and not just put in footer directly?

Comment (jacekkalbarczyk) [reply frontend/src/components/PageFooter.vue]: That's a VueJS's requirement.

File: frontend/src/components/PageFooter.vue [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        <footer>Copyright © Code for Poznan 2019</footer>
+    </div>
+</template>
+
+<script>
+export default {
```
Comment (jacekkalbarczyk) [inline frontend/src/components/PageFooter.vue]: I guess <script> section is not required if component doesn't contain JS logic. Or is it (for more elegant code styling)? To be considered.

File: frontend/src/components/PageFooter.vue [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+<style lang="scss" scoped>
+@import "./../main.scss";
+footer {
+    color: $white;
+    background-color: $blue-nav-footer;
+    padding: 25px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (jacekkalbarczyk) [inline frontend/src/components/PageFooter.vue]: I'm not sure if this line(align-items) is needed, it doesn't seem to do anything (correct me if I'm wrong).

Comment (Loczi94) [reply frontend/src/components/PageFooter.vue]: It aligns items vertically, but in this case you're right - it's not needed.

**Pull request merged**